private_demo/main.tf

```diff
@@ -15,7 +15,7 @@ terraform {
 
     cloudflare = {
       source  = "cloudflare/cloudflare"
-      version = "~> 3.0"
+      version = "~> 4.0"

```

Comment:
The private demo is being replaced with this change, which is not expected.
We need to investigate and fix that before applying here.